### ISSUE TITLE
New Tab indicator style

### ISSUE THREAD

**Issue by erneandroid** [TeamNewPipe/NewPipe#6901]
Can you implement it for tabs default view?
```xml
<?xml version="1.0" encoding="utf-8" ?>
<layer-list
	xmlns:android="http://schemas.android.com/apk/res/android">
	<item
		android:left="2dp"
		android:right="2dp">
		<shape
			android:shape="rectangle">
			<solid
				android:color="@android:color/white" />
			<corners
				android:topLeftRadius="3dp"
				android:topRightRadius="3dp"
				android:bottomLeftRadius="0dp"
				android:bottomRightRadius="0dp" />
			<size
				android:height="3dp" />
		</shape>
	</item>
</layer-list>
```

**Comment by erneandroid** [comment]
Implement this code for make tabs indicator like MD2.
Rounded in the corners and with low elevation.
Also you can replace the code on /res/drawable/mtrl_tabs_default_indicator.xml

**Comment by litetex** [comment]
@erneandroid 

I also don't understand your explanations. 
What does e.g. MD2 means? Currently I found this: https://en.wikipedia.org/wiki/MD2_(hash_function)

As for now I'm closing this issue.

Please fill out the template accordingly or (recommended) open a new issue.
You might also create a new PullRequest (however you should always open a issue first, because if it turns out that the issue won't be done you can safe yourself from doing the work).

**Comment by mhmdanas** [comment]
> What does e.g. MD2 means?

It's short for Material Design 2.
